routes/chat.js

```diff
@@ -13,7 +13,7 @@ const {
   sendSignedMessage,
   findOrCreateChannelBySignedSender
 } = require('../controllers/chat');
-const {ChatValidation} = require('../joi-validations/chat.validation');
+const {ChatValidation} = require('../joi-validations/chat.validations');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `ChatValidation` has been updated to reflect the new file name. This change seems fine as long as the new file `chat.validations.js` exists and exports `ChatValidation`.
```diff
- const {ChatValidation} = require('../joi-validations/chat.validation');
+ const {ChatValidation} = require('../joi-validations/chat.validations');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->